AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -1246,6 +1246,7 @@ imslp.org#%#AG_onLoad(function(){var b=document.getElementById("sm_dl_wait");if(
 ||cdn.web.tv/asset/javascripts/smothScroll.min.js
 ||serialuk.net/js/copyright.js
 !--- CSS ---
+jootc.com#$#body, div, p, span, h1, h2, h3, h4, h5, h6, a, img { -webkit-touch-callout: default!important; -moz-user-select: auto!important; -webkit-user-select: auto!important; -ms-user-select: auto!important; -o-user-select: auto!important; -khtml-user-select: auto!important; user-select: auto!important; }
```

Comment:
As I said before, we don't use more than one selectors. These rules should be enough
```
jootc.com#$#html > body { -webkit-touch-callout: default!important; -moz-user-select: auto!important; -webkit-user-select: auto!important; -ms-user-select: auto!important; -o-user-select: auto!important; -khtml-user-select: auto!important; user-select: auto!important; }
jootc.com#$#html > body * { -webkit-touch-callout: default!important; -moz-user-select: auto!important; -webkit-user-select: auto!important; -ms-user-select: auto!important; -o-user-select: auto!important; -khtml-user-select: auto!important; user-select: auto!important; }
```

Reply:
Okay. Thank you for your advice.
https://github.com/AdguardTeam/AdguardFilters/pull/86221/commits/339eb47f8524d7cc64ca9e06f08485eca64d034e